File: Práctica 1/Java/practica1-prueba/p05.java
//ANTONIO JOSE FERNANDEZ BELLURE

//prueba04 de las del corrector de Alicia(supuestamente)

import java.io.*;
import java.util.ArrayList;

public class p05{

	public static void main(String[] args) {

		Coleccion colec = new Coleccion();

		colec.lectura(args[0]);//este fichero tiene el mapa y diferentes localidades con la misma ubicacion

		System.out.println(colec);

		
	}
}

File: Práctica 1/C++/practica1-prueba/p05.cc
#include <iostream>
#include <vector>
#include <fstream>
#include <sstream>
#include <cstdlib>
#include "Coleccion.h"

using namespace std;

int main(int argc, char *argv[]){

	Coleccion colec = Coleccion();
	string fichero=argv[1];
	colec.lectura(fichero);

	cout<<colec<<endl;

	

	return 0;
}